src/main/java/core/basesyntax/service/impl/StrategyOperationServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.OperationType;
+import core.basesyntax.service.StrategyOperationService;
+import core.basesyntax.strategy.OperationHandler;
+import exception.OperationException;
+import java.util.Map;
+
+public class StrategyOperationServiceImpl implements StrategyOperationService {
+    private Map<OperationType, OperationHandler> operationHandlerMap;
```

Comment:
Consider making `operationHandlerMap` final to indicate that it should not be reassigned after initialization. This can help prevent accidental modifications to the map reference.